package.json

```
@@ -29,17 +29,16 @@
     "link-win": "node ./scripts/link-bin.js node_modules/amplify-cli-internal/bin/amplify amplify-dev",
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
     "package": "lerna run package",
-    "postpublish:release": "git fetch . release:main && git push origin main",
-    "postpublish:api-plugin-stable": "git push origin api-plugin-stable",
+    "postpublish:release-api-plugin-stable": "./codebuild_specs/scripts/post_publish.sh",
     "prettier-check": "yarn prettier --check .",
     "prettier-write": "yarn prettier --write .",
     "production-install": "yarn --frozen-lockfile --cache-folder ~/.cache/yarn",
     "production-build": "yarn production-install && lerna run build --concurrency 3 --stream",
-    "publish-to-verdaccio": "lerna publish --yes --no-commit-hooks --no-push --exact --dist-tag=latest --conventional-commits --no-git-tag-version --no-verify-access",
+    "publish-to-verdaccio": "lerna publish --dist-tag=api-plugin-stable --yes --no-commit-hooks --no-push --exact --conventional-commits --no-git-tag-version --no-verify-access",
     "publish:beta": "lerna publish --exact --dist-tag=beta --preid=beta --conventional-commits --conventional-prerelease --message 'chore(release): Publish [ci skip]' --no-verify-access --yes",
     "publish:main": "lerna publish --canary --force-publish --preid=alpha --exact --include-merged-tags --conventional-prerelease --no-verify-access --yes",
     "publish:release": "lerna publish --conventional-commits --conventional-graduate --exact --yes --message 'chore(release): Publish [ci skip]' --no-verify-access",
-    "publish:api-plugin-stable": "lerna publish --dist-tag=api-plugin-stable --conventional-commits --conventional-graduate --exact --yes --message 'chore(release): Publish [ci skip]' --no-verify-access",
+    "publish:release-api-plugin-stable": "lerna publish --dist-tag=api-plugin-stable --conventional-commits --conventional-graduate --exact --yes --message 'chore(release): Publish [ci skip]' --no-verify-access",
```

Nit/suggestion: can we update the message to include the version number? (Note double quotes to preserve the NPM_TAG env var. I'm assuming that's the correct var to output the new release version.)

```suggestion
    "publish:release-api-plugin-stable": "lerna publish --dist-tag=api-plugin-stable --conventional-commits --conventional-graduate --exact --yes --message \"chore(release): Publish release $NPM_TAG\" --no-verify-access",
```

---

We do not have the version number available in env variables as lerna computes the new version to publish based on conventional commit history, as part of this command. We could do a `lerna version --no-push` before we publish to get the versions that will eventually be published, but that command updates local `package.json` files with new version and we would need to revert those before we run this publish command.

---

Bummer, thanks for the explanation.